js/state/contentSettings.js

```diff
@@ -123,6 +127,10 @@ const getContentSettingsFromSiteSettings = (appState) => {
       addContentSettings(contentSettings.javascript, hostPattern, '*',
         hostSetting.noScript ? 'block' : 'allow')
     }
+    if (typeof hostSetting.allowActiveMixedContent === 'boolean') {
```

Comment:
I think it would be better to name this setting `runInsecureContext`. It's easier if we don't have to translate names across various parts of the app